services/src/main/scala/cromwell/pipeline/service/AkkaHttpClient.scala

```diff
@@ -0,0 +1,85 @@
+package cromwell.pipeline.service
+
+import java.nio.charset.Charset
+
+import akka.actor.ActorSystem
+import akka.http.scaladsl.model.headers.RawHeader
+import akka.http.scaladsl.model.{ HttpMethods, HttpRequest, HttpResponse }
+import akka.http.scaladsl.{ Http, HttpExt }
+import akka.stream.ActorMaterializer
+import akka.util.ByteString
+
+import scala.concurrent.{ ExecutionContext, Future }
+
+class AkkaHttpClient extends HttpClient {
+  private implicit val actorSystem: ActorSystem = ActorSystem()
+  private implicit val materializer: ActorMaterializer = ActorMaterializer()
+
+  def http: HttpExt = Http()
+
+  override def get(url: String, params: Map[String, String] = Map(), headers: Map[String, String] = Map())(
+    implicit ec: ExecutionContext
+  ): Future[Response] = {
+    println(queryBuilder(url, params))
+    val futureResponse: Future[HttpResponse] = http.singleRequest(
+      HttpRequest(method = HttpMethods.GET, uri = queryBuilder(url, params)).withHeaders(parseHeaders(headers))
+    )
+    responsify(futureResponse)
+  }
+
+  override def post(
+    url: String,
+    params: Map[String, String] = Map(),
+    headers: Map[String, String] = Map(),
+    payload: String = ""
```

Comment:
As we discussed in the last meeting, don't use empty string

Reply:
fixed